<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowVideoWidget</class>
 <widget class="QWidget" name="ShowVideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>304</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_back" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget_back
{
border:1px solid rgba(74, 108, 155, 0);
border-radius:2px;
	background-color: rgba(74, 108, 155, 0);
}

</string>
     </property>
     <widget class="QWidget" name="widget_name" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>371</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_name
{
	background-color: rgba(220, 255, 193, 0);
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>62</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;微软雅黑&quot;;
	color:rgb(33,33,33);
	font-size: 13px;
border:1px solid rgba(102, 102, 102, 92);
	border-radius:1px;
	
background-color: rgb(255, 255, 255);
	
	</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_erro" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>331</width>
        <height>49</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_erro
{
	background-color: rgba(220, 255, 193, 0);
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_erro">
         <property name="styleSheet">
          <string notr="true">	font-family: &quot;微软雅黑&quot;;
	color:rgb(233,233,233);
	font-size: 16px;
border:1px solid rgba(102, 102, 102, 92);
	border-radius:1px;
	padding-left:6px;
padding-right:6px;
background-color: rgb(255, 25, 25);
	
	</string>
         </property>
         <property name="text">
          <string>播放失败，重试中...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_close">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>36</width>
        <height>36</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
image: url(:/image/closebtn_3.png);
background-color:rgba(200,80,6,0);  
border-radius:18px; 
color: rgb(255, 255, 255);
font-size:16px;
font-family:&quot;微软雅黑&quot;
} 
QPushButton:hover{ 
background-color:rgb(200,80,6); 
} 
QPushButton:pressed{ 
background-color:rgb(200,80,6);  
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
